app/registries/foreman/settings/general.rb

```diff
@@ -0,0 +1,105 @@
+Foreman::SettingManager.define(:foreman) do
+  protocol = SETTINGS[:require_ssl] ? 'https' : 'http'
+  domain = SETTINGS[:domain]
+  administrator = "root@#{domain}"
+  foreman_url = "#{protocol}://#{SETTINGS[:fqdn]}"
+
+  locales = -> { Hash['' => _("Browser locale")].merge(Hash[FastGettext.human_available_locales.map { |lang| [lang[1], lang[0]] }]) }
+  timezones = -> { Hash['' => _("Browser timezone")].merge(Hash[ActiveSupport::TimeZone.all.map { |tz| [tz.name, "(GMT #{tz.formatted_offset}) #{tz.name}"] }]) }
+
+  category(:general, N_('General')) do
+    setting('administrator',
+      type: :string,
+      description: N_("The default administrator email address"),
+      default: administrator,
```

Comment:
Perhaps inline it here? It's only used once anyway.
```suggestion
      default: "root@#{SETTINGS[:domain}",
```